docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -174,3 +213,4 @@ display JP2 images in their browse clients.
 
 For more information about using querying Helioviewer.org, see the Helioviewer.org
 API documentation at: `https://api.helioviewer.org/docs/v2/ <https://api.helioviewer.org/docs/v2/>`__.
+
```

Comment:
I was thinking, we probably want to make some where in this file that our client is not a meant to offer full access to the HelioViewer API and we do not support all the inputs that the official API allows. 

But we do support the user passing in extra keywords but we make no promise that the ones outside the ones we list will work.

Reply:
Sorry, I didn't get that

Reply:
We want to make it clear to any reader/user that HelioviewerClient is not meant to offer full access to the HelioViewer API and as such we do not support all the inputs that the official API allows.

But we do allow the user to pass in extra keywords but we make no promise that the ones not documented will work.

Reply:
Oh, Ok